packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.tsx

```diff
@@ -23,8 +24,14 @@ export class DocumentCard extends React.Component<IDocumentCardProps, any> {
       };
     }
 
+    // if this element is actionable it should have an aria role
+    let role = actionable ? (onClick ? 'button' : 'link') : null;
+    let tabIndex = actionable ? 0 : null;
+
     return (
       <div
+        tabIndex={ tabIndex }
```

Comment:
@yiminwu what if we make this a tab stop only if it is actionable? If you make this card a link or a button role (which it will be if it is actionable) then it has to have a tab stop, per accessibility requirements (need to get to it w/o mouse).